desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,32 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Add Desktop Version to 'docker version' output.
+
+### Upgrades
+
+- Bump ECR Credential helper to 0.6.0
+
+### Bug fixes and minor changes
```

Comment:
All of these should be linked to public tickets (if they exist).

Reply:
Also the descriptions are a bit unclear: as a user I wouldn't necessarily know whether an issue I was seeing was the one mentioned here.